flask_resty/spec/swagger_utils.py

```diff
@@ -0,0 +1,31 @@
+import re
+
+__all__ = ['ref',
+           'flaskpath2swagger',
+           'get_marshmallow_schema_name',
+           ]
+
+RE_SWAGGER_URL = re.compile(r'<(?:[^:<>]+:)?([^<>]+)>')
+MARSHMALLOW_PLUGIN_NAME = 'apispec.ext.marshmallow'
+
+
+def ref(schema_name, path='definitions'):
+    """Generate a ref object"""
+    return {'$ref': '#/{}/{}'.format(path, schema_name)}
+
+
+def flaskpath2swagger(path):
```

Comment:
I guess you are right. I blidly copied it from https://github.com/marshmallow-code/apispec/blob/dev/apispec/ext/flask.py#L31:)